Codegen/test/resources/output/TypeCobol/LineExceed5.rdz.cbl

```diff
@@ -149,13 +125,8 @@
       *     input(param1: pic X)
            PERFORM TC-INITIALIZATIONS
```

Comment:
There are still code related to existing codegen.
This code must be located in a C# class (not in skeleton).

Can you search and adapt also this code?
Do you want an example of the result ?

Reply:
There is at least TypeCobol.Codegen.Nodes.FunctionDeclarationCG.cs to adapt

Reply:
done and add blank lines after program of procedure.
https://github.com/TypeCobolTeam/TypeCobol/pull/1672/commits/818e9394e48e3d4f16bac7de35dc44ee1c46710f

Reply:
TypeCobol.Codegen.Nodes.FunctionDeclarationCG.cs still need to be adapted.

Reply:
You can look at test:
Codegen\test\resources\output\TypeCobol\CallPublicProcFromPrivateProc.rdz.tcbl

Reply:
The expected result of the test is:
```cob
       IDENTIFICATION DIVISION.
       PROGRAM-ID. PGM1.

       DATA DIVISION.
       Working-STORAGE SECTION.

      *01  mydate1     TYPE Date.
       01 mydate1.
           02 YYYY PIC 9(4).
           02 MM PIC 9(2).
           02 DD PIC 9(2).
                                 

       PROCEDURE DIVISION.

      *declare procedure check private
      *   input mydate        TYPE Date
      *  .

       TRAITEMENT.
      *OK  call check of PGM1
      *   call check input mydate1
          CALL 'a0508f35' USING
                                 mydate1
           end-call
                                  
      
        .

      *PersonService contains public procedure
       IDENTIFICATION DIVISION.
       PROGRAM-ID. PersonService.
       DATA DIVISION.
                                                                  
       WORKING-STORAGE SECTION.

       
       LINKAGE SECTION.
       01 TC-FunctionCode pic X(30).
      * Function which call program cd991005
      * Which is generated code for PersonService.GetPersonById
           88 Fct-cd991005-GetPersonById
              value 'Fct=cd991005-GetPersonById'.

       01 arg1 pic X.
       PROCEDURE DIVISION USING TC-FunctionCode
                           arg1.
           PERFORM INIT-LIBRARY
           PERFORM FctList-Process-Mode
           GOBACK.

       FctList-Process-Mode.
           evaluate true
              when Fct-cd991005-GetPersonById
                 call 'cd991005' using arg1
              when other
                 TODO
           end-evaluate
           .
                                 
      *declare procedure GetPersonById public
      *   input  personId  type date.
       END PROGRAM PersonService.


       END PROGRAM PGM1.

      *
      *declare procedure GetPersonById public
      *   input  personId  type date.
      *_________________________________________________________________
       IDENTIFICATION DIVISION.
       PROGRAM-ID. cd991005.
       DATA DIVISION.
       LINKAGE SECTION.
      *PGM1.GetPersonById - Params :
      *     input(personId: DATE)
       01 personId.
           02 YYYY PIC 9(4).
           02 MM PIC 9(2).
           02 DD PIC 9(2).
       PROCEDURE DIVISION
             USING BY REFERENCE personId
           .
      *PGM1.GetPersonById - Params :
      *     input(personId: DATE)
           CONTINUE.
       END PROGRAM cd991005.


      *
      *declare procedure check private
      *   input mydate        TYPE Date
      *  .
      *_________________________________________________________________
       IDENTIFICATION DIVISION.
       PROGRAM-ID. a0508f35.
       data division.
       working-storage section.
      *PGM1.check - Params :
      *     input(mydate: DATE)
       
       01 TypeCobol-Generated.
         05 TC-PersonSe pic X(08) value 'PERSONSE'.
         05 PersonSe-Fct-cd991005-GetPerso pic X(30)
             value 'Fct=cd991005-GetPersonById'.
      
       linkage section.
      *PGM1.check - Params :
      *     input(mydate: DATE)
       01 mydate.
           02 YYYY PIC 9(4).
           02 MM PIC 9(2).
           02 DD PIC 9(2).
       PROCEDURE DIVISION
             USING BY REFERENCE mydate
           .
      *PGM1.check - Params :
      *     input(mydate: DATE)
      *    Call PersonService::GetPersonById input mydate
           CALL 'zcallpgm' using TC-PersonSe
                    PersonSe-Fct-cd991005-GetPerso
                                 mydate
           end-call
                                                         
           .
       END PROGRAM a0508f35.


```

Reply:
done.
https://github.com/TypeCobolTeam/TypeCobol/pull/1672/commits/8928c8c2c550d45632df004469afa36c55002b4c